### PR TITLE
doc: spelling mistakes in various sections of the user guide V2

### PR DESCRIPTION
**Pull request by macpas**
Various spelling issues within a few sections of the suricata user guide.

- [x] I have read the contributing guide lines at https://redmine.openinfosecfoundation.org/projects/suricata/wiki/Contributing
- [x] I have signed the Open Information Security Foundation contribution agreement at https://suricata-ids.org/about/contribution-agreement/
- [x] I have updated the user guide (in doc/userguide/) to reflect the changes made (if applicable)

Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket:

Describe changes:
- spelling issues in documentation
- ensure used words are English USA

[PRScript](https://redmine.openinfosecfoundation.org/projects/suricata/wiki/PRscript) output (if applicable):

### CODE REVIEW

**Comment by victorjulien** [comment]
Think consistency would be good. Lets settle on US.

**Comment by norg** [comment]
Even in Europe most people use the US English so would agree with that decision.

File: doc/userguide/configuration/suricata-yaml.rst [hunk 1]
```diff
@@ -625,7 +625,7 @@ Detection engine
...
...h the network traffic will be compared. The fact is, that many rules certainly will not be necessary. (For instance: if there appears a packet with the UDP-protocol, all signatures for the TCP-protocol won't be needed.)  For that reason, all signatures will be divided in groups. However, a distribution containing many groups will make use of a lot of memory. Not every type of signature gets its own group. There is a possibility that different signatures with several properties in common, will be placed together in a group.  The quantity of groups will determine the balance between memory and performance. A small amount of groups will lower the performance yet uses little memory. The opposite counts for a higher amount of groups. The engine allows you to manage the balance between memory and performance. To manage this, (by determining the amount of groups) there are several general options:high for good performance and more use of memory, low for low performance and little use of memory. The option medium is the balance between performance and memory usage. This is the default setting. The option custom is for advanced users. This option has values which can be managed by the user.
```
Comment (norg) [inline doc/userguide/configuration/suricata-yaml.rst]: Before "For that reason" and "The quantity of groups" are double spaces, we could use this PR to fix this as well :) Also after "general options:" we want a space before the "high for good.."

Comment (macpas) [reply doc/userguide/configuration/suricata-yaml.rst]: OK I will do it; for touched files here, check double space.

**Comment by macpas** [comment]
replaced by PR https://github.com/OISF/suricata/pull/3379

**Pull request closed without merging**